src/features/lightspeed/utils/uiUtils.ts

```diff
@@ -0,0 +1,11 @@
+export function getModelDetailsString(
+  modelIdOverride: string | undefined,
+  currentModelId: string | undefined
+): string {
+  const orgDefault = "(Red Hat org default)";
```

Comment:
Can move the constant here https://github.com/ansible/vscode-ansible/blob/main/src/definitions/lightspeed.ts
Do we need `Red Hat` in the name? This might be confusing as different org can have different base model.
How about just `default` to start with and `Model Id (default)` when extension receives the first LS response.
So the hover display would be
```
Model: default
```
```
Model: b8a86296-ef64-4ddb-bbf4-a2fd164577bb<|sepofid|>granite-3b (default)

Reply:
I have moved the string definition to lightspeed.ts.

On the model ID display for default one, I have modified the code following suggestions from Marty and Robin and dropped the code to show the actual ID.  So when the default model is used, always `Model: default` will be shown.